legate/setup.py

```diff
@@ -21,23 +20,7 @@
 
 
 setup(
-    name="legate_kvikio",
-    version=versioneer.get_version(),
-    description="KvikIO - GPUDirect Storage",
-    url="https://github.com/rapidsai/kvikio",
-    author="NVIDIA Corporation",
-    license="Apache 2.0",
-    classifiers=[
-        "Intended Audience :: Developers",
-        "Topic :: Database",
-        "Topic :: Scientific/Engineering",
-        "License :: OSI Approved :: Apache Software License",
-        "Programming Language :: Python",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
-    ],
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
```

Comment:
Not critical since kvikio doesn't have wheels, but scikit-build has bugs with `include_package_data` so specifying `package_data` explicitly (like in the non-legate setup.py) is safer. That said I don't see wheels happening for legate-kvikio anytime soon so it's mostly just to be safe.

Reply:
#233.

Reply:
I'll unresolve this thread -- I'd be happy to try and fix #233 in this PR.